### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by nisprateek** [OpenMined/courses#60] [Introduce Yourself]
Hi all!

I am Nishant. I studied computer science and computational linguistics at IIIT Hyderabad. My past research work has been mostly focused on NLP and speech processing. More recently I had been working on explicit density estimation techniques (like variational autoencoders and normalizing flows) as applied to speech synthesis.

I have recently joined the newly-formed AI Research and POC group at Citi in their London Innovation Lab where I'll be looking at a variety of research themes. Before this, I have worked in the text-to-speech research for Amazon Alexa and AWS Polly, and the speech and intent group for Cortana and Skype at Microsoft.

Through this course I aim to better understand privacy preserving machine learning methods and apply those to real-world problems. I am looking forward to learning from all of you, contributing to projects at OpenMined, and possibly publishing some interesting papers.

**Comment by yemikifouly** [comment]
Hi Nishant! That is one impressive journey and resume that you have there! 🙂 I am sure that you will enjoy the courses. For anyone involved in AI research and using data, privacy is a concern and I am confident that in the courses you will get answers to questions you had and even some you hadn't thought of. 🙂 Welcome Nishant!
